tensorboard/webapp/metrics/views/metrics_views_module.ts

```diff
@@ -24,6 +25,12 @@ import {RightPaneModule} from './right_pane/right_pane_module';
 @NgModule({
   declarations: [MetricsDashboardContainer],
   exports: [MetricsDashboardContainer],
-  imports: [CommonModule, RightPaneModule, RunsSelectorModule, MainViewModule],
+  imports: [
+    CommonModule,
+    RightPaneModule,
+    RunsSelectorModule,
+    MainViewModule,
+    FeatureFlagModule,
```

Comment:
Given that the only view code that seems to use feature flags is right-pane-related and that RightPaneModule imports FeatureFlagModule, should we actually modify MetricsViewsModule to also import FeatureFlagsModule?